src/app/core/session/session-service/session-manager.service.ts

```diff
@@ -113,8 +113,8 @@ export class SessionManagerService {
     const entityType = Entity.extractTypeFromId(entityId);
     this.entityMapper
       .load(entityType, entityId)
-      .then((res) => this.currentUser.next(res))
-      .catch(() => undefined);
+      .catch(() => undefined)
+      .then((res) => this.currentUser.next(res));
```

Comment:
The difference is that in the new order the `then()` block is always executed, in case of an error  here setting our `currentUser` to undefined (which is intended).

See fiddle for a reproduction of the Promise behaviour: https://jsfiddle.net/p6ezLmc7/1/